test/tests/error-handling-prod.test.js

```diff
@@ -166,22 +164,24 @@ describe('graphql - error handling in production', () => {
       expect(response.data.errors[0].extensions.details[0]).not.toHaveProperty('stacktrace') // No stacktrace in production
       expect(response.data.errors[0].extensions.details[1]).not.toHaveProperty('stacktrace') // No stacktrace in production
       const log = console.warn.mock.calls[0][1] || JSON.parse(console.warn.mock.calls[0][0])
-      const msgProperty = log.msg ? 'msg' : 'message'
 
       if (cds.env.features.cds_assert) {
-        // REVISIT: message missing, cds.assert also misses some properties like type, element, entity ...
```

Comment:
those properties are not needed. as long as there is a target, the developer can look up the rest.